GTG/gtk/editor/recurring_menu.py

```diff
@@ -92,25 +90,26 @@ def update_task(self, enable=True):
 
     def update_header(self):
         """ Updates the header anytime a term is selected """
-        formated_term = self.selected_recurring_term
         if self.is_term_set():
-            if formated_term.isdigit():
-                if len(formated_term) <= 2 :
-                    day = datetime.strptime(f'{formated_term}', '%d').strftime('%d')
-                    formated_term = _('{day} of the Month')
-                    formated_term = formated_term.format(day=day)
-                else:
-                    formated_term = datetime.strptime(f'{formated_term[:2:]}-{formated_term[2::]}', '%m-%d').strftime('%B %d')
-            elif formated_term == 'week':
-                formated_term = datetime.today().strftime('%A')
-            elif formated_term == 'month':
-                formated_term = _('{day} of the Month')
-                formated_term = formated_term.format(day=datetime.today().strftime('%d'))
-            elif formated_term == 'year':
-                formated_term = datetime.today().strftime('%B %d')
+            if self.selected_recurring_term.isdigit():
+                if len(self.selected_recurring_term) <= 2 : # Recurring monthly from selected date
+                    self.title.set_markup(_('Every <b>{month_day} of the month</b>').format(month_day=datetime.strptime(f'{self.selected_recurring_term}', '%d').strftime('%d')))
+                else: # Recurring yearly from selected date
+                    self.title.set_markup(_('Every <b>{year_date}</b>').format(year_date=datetime.strptime(f'{self.selected_recurring_term[:2:]}-{self.selected_recurring_term[2::]}', '%m-%d').strftime('%B %d')))
```

Comment:
I would do something like `_('Every <b>{month} {day}</b>')` since in german, we usually write the date in "day month year" rather than "month day year", and the current solution doesn't allow to change the order.